files/q2.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+def sizeOfFiles(path):
+    totalSize = 0
+    return os.path.getsize(path)
```

Comment:
I was expecting more of checking individual files and subfolders etc but if this works then fair enough. Job Done!

Reply:
I think this may only work on individual files